microsoft/PowerToys
# [installer] Support per-user installation

## Summary of the Pull Request


## PR Checklist

- [x] **Closes:** #15516 #15517


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed
 - build both installers
 - try installing both versions of PT 
   - ensure that per-machine version is installed in Program files by default and registry entries are in HKLM
   - ensure that per-user version is installed in user's app data folder by default and registry entries are in HKCU
   - try installing per-user version over per-machine version
     - ensure that error message that per-machine version is already installed is shown
   - try installing per-machine version over per-user version
     - ensure it works
